ocrd_network/ocrd_network/processing_worker.py

```diff
@@ -44,7 +44,7 @@
 
 class ProcessingWorker:
     def __init__(self, rabbitmq_addr, mongodb_addr, processor_name, ocrd_tool: dict, processor_class=None) -> None:
-        self.log = getLogger(__name__)
+        self.log = getLogger('ocrd_network.processing_worker')
         # TODO: Provide more flexibility for configuring file logging (i.e. via ENV variables)
         file_handler = logging.FileHandler(f'/tmp/worker_{processor_name}_{getpid()}.log', mode='a')
```

Comment:
How could we create that in the configuration file when a `processor_name` and `pid` is involved? For other modules such as the Processing Server or Server Cache, this is not an issue and the pattern with the configuration file could be followed. However, for the dynamic creation of log files, it will be a pain to manually put things in the log config file.

Reply:
You don't have to put every logger you're using in the config file, only in cases where the logger's level deviates from the ancestor in the logger hierarchy.

```
getLogger(f'ocrd_network.worker.{processor_name}.{pid}')
```

As it is currently implemented, this would mean that it should inherit the `DEBUG` level from

```conf
[logger_ocrd_network]                          
level=DEBUG                                    
qualname=ocrd_network                          
```

But you could override the `ocrd_network.worker` logger in the config file to change the log level to something else.

Reply:
Got it, that makes sense, yes.